lib/ui/write-error.js

```diff
@@ -4,8 +4,9 @@ var chalk = require('chalk');
 module.exports = function writeError(ui, error){
   if (!error) { return; }
 
-  if (error.file) {
-    var file = error.file;
+  // Uglify errors have a filename instead
+  if (error.file || error.filename) {
```

Comment:
Can you do `var fileName = error.file || error.fileName;` and then the conditional?